packages/contentstack-import/src/import/modules-js/locales.js

```diff
@@ -127,6 +127,15 @@ module.exports = class ImportLanguages {
         .catch(function (error) {
           // error while importing languages
           fileHelper.writeFileSync(langFailsPath, self.fails);
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).locale !== undefined) {
+            if (JSON.parse(error.request.data).locale.code) {
+              message_content_type =
+                ' For language with code - ' + JSON.parse(error.request.data).locale.code;
```

Comment:
@cs-raj Can you please validate the error messages with the docs team?

Reply:
Validated with Varsha and made the changes